Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -137,9 +142,9 @@ bool CBoot::EmulatedBS2_GC()
 
 		RunFunction(iAppLoaderMain);
 
-		u32 iRamAddress = Memory::ReadUnchecked_U32(0x81300004);
-		u32 iLength     = Memory::ReadUnchecked_U32(0x81300008);
-		u32 iDVDOffset  = Memory::ReadUnchecked_U32(0x8130000c);
+		u32 iRamAddress = PowerPC::Read_U32(0x81300004);
```

Comment:
What determines which of these addresses we change to physical, and which ones we leave as BAT addresses? Not objecting, just kind of want to know